time last_update
string landmark_id
geometry_msgs/Point position
float64[9] position_covariance
